.github/workflows/tag-release.yml

```diff
@@ -18,20 +12,28 @@ jobs:
         GITHUB_CONTEXT: ${{ toJson(github) }}
       run: echo "$GITHUB_CONTEXT"
       shell: bash
+
     - name: Check out code
       uses: actions/checkout@v3
+
+    - name: Compute new version number
+      run: |
+        versionNumber=$(git tag -l --sort=version:refname | grep -v '-' | tail -1 | awk '{print $2 + 1}' FS='.')
```

Comment:
how does that FS bit work?  Is that a interpreted as a parameter to awk?

Reply:
Yeah, it's the regex on which to split the string.